docker-compose-win.yml

```diff
@@ -0,0 +1,56 @@
+# before run use command `docker volume create postgres`
+# run docker-compose -f docker-compose-win.yml up
+version: "3"
+services:
+  web:
+    build: .
+    container_name: web01
```

Comment:
Why do you add `01` suffix to everything?

Reply:
It's one of our futures swarm :)